### PR TITLE
serialblaster bugfix

### PR DESCRIPTION
**Pull request by adamkaliszan**
## Summary
Serlalblaster exemplary app bugfixes.

In my opinion there ware many bugs in previous implementation. 
## Impact
None
## Testing
Yes, on Nucleo Wl55JC board

### CODE REVIEW

File: examples/serialblaster/serialblaster_main.c [hunk 1]
```diff
@@ -59,7 +59,7 @@ int main(int argc, FAR char *argv[])
   int ret;
   int fd;
   FAR char *devpath;
-  int size = 0;
+  int size = sizeof(s)-1;
```
Comment (davids5) [inline examples/serialblaster/serialblaster_main.c:62]:
```suggestion
  const int slength  = sizeof(s)-1;
  int size = slength;
```
and replace it throughout the code

**Review by davids5 — CHANGES_REQUESTED** [review]
Might be better to clean it up in this way

File: examples/serialblaster/serialblaster_main.c (hunk 1 above)
Comment (adamkaliszan) [reply examples/serialblaster/serialblaster_main.c:62]: OK. I wanted to reduce number of changes. sizeof(s)-1 in many places is annoying.

File: examples/serialblaster/serialblaster_main.c [hunk 2]
```diff
@@ -84,25 +85,25 @@ int main(int argc, FAR char *argv[])
   fd = open(devpath, O_RDWR);
   if (fd < 0)
     {
-      printf("dev_ttyS2: ERROR Failed to open /dev/ttyS2\n");
+      printf("%s: ERROR Failed to open %s\n", devpath, devpath);
```
Comment (pkarashchenko) [inline examples/serialblaster/serialblaster_main.c:88]: Why do we need to print `devpath` two times?

Comment (adamkaliszan) [reply examples/serialblaster/serialblaster_main.c]: OK. It is done. Now there is argv[0] and devpath

File: examples/serialblaster/serialblaster_main.c (hunk 1 above)
Comment (adamkaliszan) [reply examples/serialblaster/serialblaster_main.c]: It is done

**Review by pkarashchenko — APPROVED** [review]

**Pull request merged**